src/haxeLanguageServer/helper/DocHelper.hx

```diff
@@ -109,7 +109,13 @@ class DocHelper {
 		return '```$languageId\n$content\n```';
 	}
 
-	public static function patchRange(doc:TextDocument, range:Range) {
+	/**
+		expands range to encompass full lines when range has leading or trailing whitespace in first and / or last line
+
+		@param doc
```

Comment:
Don't leave empty auto-generated `@param` tags in the code please. ;) (there should be a checkstyle check for that...)

Reply:
there is, but project doesn't use checkstyle ;)